gauss_jordan.go

```diff
@@ -89,8 +89,9 @@ func GaussJordan(linearSystem [][]*Int, linearSystemResult []*Int) ([]*Int, erro
 	current_column := make([]*Int, len(extendedMatrix))
 
 	for (index_current_row < nrows) && (index_current_column < ncols) {
-		/* Find the k-th pivot: */
+		// Find the k-th pivot:
 		current_column = ExtractColumn(extendedMatrix, index_current_column)
+		prime := current_column[0].Base
```

Comment:
Didn't see this in my previous review: the namings should respect the Go convention (this is the Python convention). Use camelCase instead.